app/servers.go

```diff
@@ -63,6 +63,8 @@ func (app *CortezaApp) mountHttpRoutes(r chi.Router) {
 			r.Route("/automation", automationRest.MountRoutes)
 			r.Route("/compose", composeRest.MountRoutes)
 
+			app.WsServer.MountRoutes(r)
```

Comment:
Can we move the "/websocket" route root here so that we follow the same pattern of route definitions?